DBM-Core/DBM-RangeCheck.lua

```diff
@@ -654,7 +663,7 @@ do
 
 	function updateIcon()
 		local numPlayers = GetNumGroupMembers()
-		activeDots = max(numPlayers, activeDots)
+		activeDots = max(numPlayers or 0, activeDots)
```

Comment:
GetNumGroupMembers always returns a number? I see no documented occurances of it being nilable.

Reply:
unfortunately the Blizzard docs are missing this function and quite some other functions :(

I need to figure out how the type annotations on the wiki work to fix this upstream, see https://github.com/Ketho/vscode-wow-api/issues/124